db/seeds.rb

```diff
@@ -12,8 +12,8 @@ def load_data
     load_crops
     load_plant_parts
 
-    # for development environments only
-    if Rails.env.development?
+    # for development and staging environments only
+    if Rails.env.development? || Rails.env.staging?
```

Comment:
This would create test1, test2, wrangler1 and admin1 users on staging with known-in-advance passwords? That doesn't sound right. I think we *should* run `load_paid_account_types` on staging, though.